Library/Homebrew/utils/formatter/columns.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/tty"
+
+module Formatter
```

Comment:
Just stick this inside the existing Formatter class; I dislike objects across multiple files